uint8 status
Header header
float64 x1
float64 y1
float64 c1
float64 x2
float64 y2
float64 c2
float64 x3
float64 y3
float64 c3
float64 xmin
float64 xmax
float64 ymin
float64 ymax
float64 highLevTime
float64 term_xmin
float64 term_xmax
float64 term_ymin
float64 term_ymax
